ra-optimisation/rao-api/pom.xml

```diff
@@ -104,5 +109,11 @@
             <scope>test</scope>
             <type>test-jar</type>
         </dependency>
+        <dependency>
+            <groupId>${project.groupId}</groupId>
+            <artifactId>farao-crac-result-extensions</artifactId>
+            <version>${project.version}</version>
+            <scope>compile</scope>
```

Comment:
not need to put compile here, it is the default value anway

should be put before the test dependencies

Reply:
removed (it was redundant actually)